apps/passport-server/src/services/generic-issuance/pipelines/LemonadePipeline.ts

```diff
@@ -1680,10 +1674,28 @@ export class LemonadePipeline implements BasePipeline {
               );
             }
 
-            // We found a Lemonade atom, so check in with the Lemonade backend
-            return this.lemonadeCheckin(ticketAtom, emailClaim.emailAddress);
+            if (ticketAtom.lemonadeUserId) {
```

Comment:
should we use manual check-in for all lemonade tix to prevent the problem you mentioned? can also be done in a follow-up pr if it's a bit more involved